.cypress/integration/ad/dashboard/ad_dashboard.spec.ts

```diff
@@ -13,31 +13,67 @@
  * permissions and limitations under the License.
  */
 
-/// <reference types="cypress" />
-
 import {
-  AD_PATH,
   AD_URL,
-  API_URL_PREFIX,
   APP_URL_PREFIX,
   DASHBOARD,
-  DETECTORS,
   SLASH,
 } from '../../../utils/constants';
 
 context('AD Dashboard', () => {
-  it('Empty dashboard - no detector', () => {
-    cy.server();
-    cy.route(
-      'GET',
-      [API_URL_PREFIX, AD_PATH, DETECTORS + '*'].join(SLASH),
-      'fixture:no_detector_index_response.json'
-    ).as('getDetectors');
+  it('Empty dashboard - no detector index', () => {
+    cy.mockGetDetectorOnAction('no_detector_index_response.json', () => {
+      cy.visit([APP_URL_PREFIX, AD_URL, DASHBOARD].join(SLASH));
```

Comment:
I wonder if we should add constants for the page url at the top of the file, so we don't have to repeatedly write `[APP_URL_PREFIX, AD_URL, DASHBOARD].join(SLASH)` all the time

Reply:
In fact, the same thing comes up in the `commands` file. Maybe move all of these to some url constants somewhere? Then we would have less constants to import as well.

Reply:
good point. use separate method to get URLs instead.